vocabularies/Hierarchy.json

```diff
@@ -33,8 +33,43 @@
       "@Core.Description": "Hierarchy-specific information in the result set of a hierarchical request",
       "@Core.LongDescription": "The [base term](https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Aggregation.V1.html#RecursiveHierarchy)\n          governs what are the nodes and parents in the hierarchy, whereas this term defines derived information."
     },
+    "HierarchyType": {
+      "$Kind": "ComplexType",
+      "@Common.Experimental": true,
+      "@Core.LongDescription": "The properties in this complex type contain information about\n          an entry in the result set of a request with multiple aggregation levels, some of which are expanded.\n          These properties also serve in the derived [`RecursiveHierarchyType`](#RecursiveHierarchyType)\n          with their definitions rephrased in the concept of recursive hierarchies.",
+      "LimitedDescendantCount": {
+        "$Type": "Edm.Int64",
+        "$Nullable": true,
+        "@Core.Description": "Number of entries from finer-grained aggregation levels that are expanded"
+      },
+      "DrillState": {
+        "$Nullable": true,
+        "@Core.Description": "Drill state of an entry",
+        "@Validation.AllowedValues": [
+          {
+            "Value": "expanded",
+            "@Core.Description": "The entry precedes entries from finer-grained aggregation levels"
+          },
+          {
+            "Value": "subtotal",
+            "@Core.Description": "The entry follows entries from finer-grained aggregation levels and contains subtotals"
+          },
```

Comment:
I don't think it's ideal that `RecursiveHierarchy` but the value `subtotal` is not allowed in the `RecursiveHierarchy`.
Also the wording of the drill states is quite different here (although the "visual interpretation" on the UI is actually the same).